### DISCUSSION TITLE
get the width and lineheight of the character

### DISCUSSION THREAD

**Discussion by zundaren** [sourcelair/xterm.js#4423] [Q&A]
I have a container 800px wide and 800px high, how do I set the appropriate rows and cols to fill the container?
If lineheight*rows != height, the background color of xterm-screen should be filled

**Comment by Tyriar** [comment]
Your request is covered in https://github.com/xtermjs/xterm.js/issues/702. The standard way to do this is to use the builtin `fit` addon, here's example usage in the demo:

https://github.com/xtermjs/xterm.js/blob/9154831afa9aea78306d8edad8fe403616758ad3/demo/client.ts#L281
